.github/Pull_request_Template.md

```diff
@@ -0,0 +1,14 @@
+# Related Issuse or bug
+  - Info about Issue or bug
```

Comment:
Please Add a line here `Fixes: #[issue number that will be closed through this PR]`

Reply:
Added!